app/controllers/availabilities_controller.rb

```diff
@@ -0,0 +1,23 @@
+class AvailabilitiesController < ApplicationController
+  def new
+    @availability = Availability.new
+  end
+
+  def create
+    @offer = current_user.offers.find_by(event_id: params[:event_id])
+    @event = @offer.event
+    @availabilities = availability_params[:available].each_index { |i| @offer.availabilities.create(available: availability_params[:available][i], call_id: availability_params[:call_id][i]) }
+    if !@availabilities.any? { |a| "true" }
```

Comment:
Unused block argument - `a`. You can omit the argument if you don't care about it.